src/pybind/mgr/smb/module.py

```diff
@@ -1,7 +1,7 @@
-from dataclasses import replace
 from typing import TYPE_CHECKING, Any, List, Optional, cast
 
 import logging
+from dataclasses import replace
```

Comment:
Please move this addition (fix, I presume) into a more appropriate patch, thanks.

Reply:
In case it's not obvious here on github ... this is currently part of a patch adding documentation. It should go with the patch updating module.py.